tasks/compile-stylesheets.mjs

```diff
@@ -135,6 +142,48 @@ export function getPathByDestination (filePath) {
     : `${name}.scss`)
 }
 
+/**
+ * Sass custom logger
+ *
+ * @returns {import('sass-embedded').Logger} Sass logger
+ */
+export const logger = () => ({
```

Comment:
Going through commit-by-commit, I'm a bit confused by this. Are we doing this specifically to enable the next commit (suppressing known deprecations and getting alerted to new ones)?

Reply:
Yeah exactly that, I've recreated the Sass logger in JavaScript

But I wish we could import + wrap the native one instead:

```mjs
import sass from 'sass-embedded'

export const logger = {
  warn (message, options) {
  
    // 1. Skip logging for known deprecations
    if (message.includes('example')) {
      return
    }

    // 2. Defer remaining messages to Sass logger
    sass.logger.warn(message, options)
  }
}
```

Instead we had to write 2) ourselves